### DISCUSSION TITLE
assertEquals in moose/test/tests/outputs/format/test_json.py should be assertEqual

### DISCUSSION THREAD

**Discussion by JohnBussoletti** [idaholab/moose#27992] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (avoid screenshots if possible, triple back quotes before/after pasted text, etc)

### Issue or question about MOOSE

Execution of run_tests reported one failure in outputs/format/test_json.py due to an entry "assertEquals" instead of "assertEqual".  A minor bug.

### Diagnostics for MOOSE installation issues

[Optional] Output of the diagnostics scripts. To run the script: cd ~/projects/moose/scripts; ./diagnostics.sh

**Comment by GiudGiud** [comment]
Hello

Do you have the log for the test? The one from running the test with run_tests
I just ran this locally and it was fine, so I'm thinking maybe a python package version issue.

Guillaume

**Comment by JohnBussoletti** [comment]
Guillaume,

It could be. Also, possibly a Conda issue.   I don’t recall which line I corrected in the Python script to correct the error.  Unfortunately, once I corrected the script, the log is clean.

Note that it was from an x86_64 build.

John

**Reply by milljm** [reply]
```bash
❯ cd ~/projects/moose
❯ git grep "assertEquals"
test/tests/outputs/format/test_json.py:        self.assertEquals(["FunctionName"], f["associated_types"])
```
Yes, this should be `self.assertEqual`.

Thank you for reporting!
Issue created: https://github.com/idaholab/moose/issues/27995
